libs/dotcms-webcomponents/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dotcms-webcomponents",
-    "version": "22.3.0-next.7",
+    "version": "22.3.0-rc.3",
```

Comment:
are these version correct ?

Reply:
no, @dsilvam this was changed on march 14

Reply:
The version in master should be kept